script.js

```diff
@@ -0,0 +1,53 @@
+console.log("Long, long ago in a galaxy far, far away... ")
+function CharacterSelect() {
+  var characterSelect = prompt ("Welcome to Star Wars: The Adventure Game! Would you like to play as (L)uke Skywalker, (H)an Solo or Lando (C)arlissian? L/H/C? (case sensitive)")
```

Comment:
how could you control for case sensitivity in case a user ignores your directions?